Makefile

```diff
@@ -82,7 +82,7 @@ ci-cs: prerequisites
 test: phpunit analyze composer-validate
 
 .PHONY: composer-validate
-composer-validate: test-prerequisites
+composer-validate:
```

Comment:
Composer should validate what we have in the repo, not generated files

Reply:
This is exactly what I don't want because `composer validate --strict` will fail if the lock file is out of date. And we don't have a lock file unless we do `composer install` or like that.

Reply:
If we do `composer update` the lock file will **always** be okay!

Reply:
Exactly! But if we mess up the Makefile, and we don't do `composer update` because of a mistake, there's no way to see this! Please put it back.